### ISSUE TITLE
md-select violates CSP

### ISSUE THREAD

**Issue by Frank3K** [angular/material#6819]
The `md-select` directive violates CSP when an item is clicked and the select closes. Upon closing, there is still some remaining styling form the ink-ripple which is applied when the item is clicked. The following HTML is added to the DOM (new lines added for readability):

``` html
<div class="md-text ng-binding">some text value</div>
<div class="md-ripple-container" style="background-color: rgba(0, 0, 0, 0.0980392);">
<div class="md-ripple md-ripple-placed md-ripple-scaled md-ripple-active" style="left: 52px; top: 22px; width: 147.472px; height: 147.472px; border-color: rgb(0, 0, 0); background: rgb(0, 0, 0);">
</div>
</div>
```

Violating call in [select.js](https://github.com/angular/material/blob/de5237f1b7ad8185e1af143f467fae01edaa33e5/src/components/select/select.js#L255).

References #6602, #6691, #6756.

**Comment by Emeegeemee** [comment]
Good catch, looking at the code, it looks like the the ripple container is being copied into the md-select-value element. This doesn't seem to be on purpose since I can't get the copied ripple container to ripple in the md-select-value. So it should be as easy as only copying the .md-text outerHTML instead of the md-option innerHTML.

**Comment by Frank3K** [comment]
Fixed via b707375.
